x-pack/metricbeat/magefile.go

```diff
@@ -28,12 +28,6 @@ func init() {
 	devtools.BeatLicense = "Elastic License"
 }
 
-// Aliases provides compatibility with CI while we transition all Beats
-// to having common testing targets.
-var Aliases = map[string]interface{}{
-	"goTestUnit": GoUnitTest, // dev-tools/jenkins_ci.ps1 uses this.
-}
-
```

Comment:
@kvch was this code removed by mistake? Windows builds are failing with `Unknown target specified: goTestUnit`. I have opened PR to recover it https://github.com/elastic/beats/pull/13190